bots/pr/src/main/java/org/openjdk/skara/bots/pr/CensusInstance.java

```diff
@@ -158,3 +161,33 @@ boolean isReviewer(HostUser hostUser) {
         return project.isReviewer(contributor.username(), version);
     }
 }
+
+class LimitedCensusInstance {
```

Comment:
Maybe move this to its own file? I try to avoid having more than one top level class in the same file, but do whatever makes the most sense (maybe Skara uses this in other places?).

Reply:
I will think about it. This is a rather common pattern in Skara, which is why I went for it.